registration.php

```diff
@@ -4,3 +4,7 @@
     'Bread_BreadCheckout',
     __DIR__
 );
+
+use Bread\BreadCheckout\Log\SentryLogger;
+
+SentryLogger::init();
```

Comment:
Why is sentry initialized here?

Reply:
Is there a better location you think? I just wanted it to be somewhere that is hit every time at the start of everything, so if you know a better place then yeah we should definitely do that

Reply:
We already have logger class, that would be a good place.